src/Database/Migration/Migrate.php

```diff
@@ -35,15 +29,14 @@ public function __construct(protected SchemaBuilder $schema, protected Applicati
 
     /**
      * Run a migration
-     *
-     * @param string $type (up|down)
      */
     public function run(
         string $path,
-        string $type = self::UP,
+        Direction $direction = Direction::UP,
         bool $oneStep = false,
         bool $forceMigration = false
-    ): void {
+    ): void
```

Comment:
Can you run the code style checkers on this PR?

Reply:
fixed